charts/opensearch/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.0.7
+version: 1.0.8
```

Comment:
We probably need another bump as another PR is already planning to 1.3.0 or over.
But we can wait for that until @DandyDeveloper @TheAlgo review.